src/gateway/environments/jsdom-sixteen/jsdom-sixteen-resource-loader.js

```diff
@@ -162,10 +162,13 @@ export class JSDOMSixteenResourceLoader extends ResourceLoader {
             agent: this._getAgent(url),
         });
         const handleInactive = abortableFetch.then((response) => {
-            if (!this._active) {
-                logger.silly(
-                    `File requested but never used: ${readableURLForLogging}`,
-                );
+            const aborted = abortableFetch.aborted;
+            if (!this._active || aborted) {
+                if (!aborted) {
+                    logger.silly(
+                        `File requested but never used: ${readableURLForLogging}`,
```

Comment:
This would be useful for us to investigate these, I suspect - since these could imply a problem happening somewhere in our code!

Reply:
When you say "these", I presume you mean the requests that were made but weren't needed to produce a result, rather than the aborted calls? Or are you suggesting a change to always log, even when requests are aborted?

Reply:
I was interested in the calls that were made but weren't needed.

Reply:
Yeah, looking at the logs may be interesting there.